wordpress-mobile/WordPress-Android
# [Gutenberg] Release v1.0.2

This PR tracks and brings in the v1.0.2 release of Gutenberg-mobile

### To test:

* Open a gutenberg post and verify that the editor opens the post normally

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`..txt`.
